File: dentofficemobile/lib/models/ocjene.dart
// -- cocjene.dart --
import 'package:decimal/decimal.dart';
import 'package:json_annotation/json_annotation.dart';

import 'korisnik.dart';

part'ocjene.g.dart';

@JsonSerializable()
class Ocjene {
  int? ocjenaId;
  int korisnikId;
  int pacijentId;
  DateTime kreirano;
  Decimal ocjena;
  String komentar;
  Korisnik? korisnik;

  Ocjene({this.ocjenaId, required this.korisnikId, required this.pacijentId, required this.kreirano,
    required this.ocjena,required this.komentar, this.korisnik});


  factory Ocjene.fromJson(Map<String, dynamic> json) => _$OcjeneFromJson(json);

  Map<String, dynamic> toJson() => _$OcjeneToJson(this);
}

File: DentOffice.Model/Ocjene.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DentOffice.Model
{
    public class Ocjene
    {
        public int OcjenaId { get; set; }

        public int PacijentId { get; set; }
        public Pacijent Pacijent { get; set; }
        public int KorisnikId { get; set; }
        public Korisnik Stomatolog { get; set; }
        public DateTime Kreirano { get; set; }
        public decimal Ocjena { get; set; }
        public string Komentar { get; set; }
    }
}
